scheduler/src/cook/mesos.clj

```diff
@@ -241,8 +239,7 @@
                                     (mesomatic.scheduler/start! driver)
                                     (reset! current-driver driver)
 
-                                    (when riemann-host
-                                      (cook.mesos.monitor/riemann-reporter mesos-datomic-conn :riemann-host riemann-host :riemann-port riemann-port))
+                                    (cook.mesos.monitor/start-collecting-stats mesos-datomic-conn)
```

Comment:
Should we make the interval configurable?

Reply:
Done